RELEASE.md

```diff
@@ -1,39 +1,39 @@
 
-The format of the release tag is either "Release-Name-RC1" or "Release-Name" depending on whether the tag is a release candidate or a release. (Example: 0.7.0-RC1 is a GitHub release tag for zfs-localpv release candidate build. 0.7.0 is the release tag that is created after the release criteria are satisfied by the zfs-localpv builds.)
+The format of the release tag follows semver versioning. The final release tag is of format `X.Y.Z` and the respective prerelease and develop tags are `X.Y.Z-prerelease` and `X.Y+1.0-develop`.
 
-Once the release is triggered, Travis build process has to be monitored. Once Travis build passes, images are pushed to docker hub and quay.io. Images can be verified by going through docker hub and quay.io. Also, the images shouldn't have any high level vulnerabilities.
+Once the release is triggered, the freezed code undergoes stages as such linting, unit-tests and bdd-tests and the code coverage is updated accordingly. Post the former jobs, the image build is triggered with the specified tag, the chart is run though scripts that update the tags at places whereever deemed necessary and eventually publish the images and respective helm charts.
 
-Images are published at following location :
+The helm charts are hosted on github deployments for the corresponding releases.
+
+Images and Helm charts are published at following location :
 
-https://quay.io/repository/openebs/zfs-driver?tab=tags
 https://hub.docker.com/r/openebs/zfs-driver/tags
+https://github.com/openebs/zfs-localpv/tree/gh-pages
 
-Once a release is created, update the release description with the changelog mentioned in `changelog/v0.7.x`. Once the changelogs are updated in the release, the repo owner needs to create a PR to `develop` with the following details:
-1. update the changelog from `changelog/v0.7.x` to `zfs-localpv/CHANGELOG.md`
-2. If a release is not an RC tag then PR should include the changes to remove `changelog/v0.7.x` folder.
-3. If a release is an RC tag then PR should include the changes to remove the changelog from `changelog/v0.7.x` which are already mentioned in `zfs-localpv/CHANGELOG.md` as part of step number 1.
+Once a release is created:-
+1. The repo owner updates the release page changelog with all the necessary content.
+2. The repo owner updates the [CHANGELOG.md](./CHANGELOG.md) file with the changelog for the release.
```

Comment:
Ideally the CHANGELOG.md should be updated before the release and not after. One should be able to pull in the git repo and read the change summary on the CHANGELOG.md.

Updates after release requires online version control software like github to be present for viewing the changelog.

Reply:
That's not entriely true, changelog is updated once the release it created. How does one add changelog before release?

Reply:
The release notes on the github release listing can't be created, that's true. However, the CHANGELOG.md can be updated before the release.

Reply:
Can you please give me the wording for the same.

Reply:
cc @niladrih

Reply:
Please find my suggestion on a comment above.